change/@uifabric-foundation-2020-10-16-15-41-44-rename-styling.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Should you also change the name of the file?

Reply:
Xu might know for sure but I don't think beachball is actually sensitive to the names.

Reply:
no need to change the file name :)